packages/ui-icons-lucide/src/wrapLucideIcon/props.ts

```diff
@@ -24,37 +24,97 @@
 
 /**
- * Extract color tokens from Icon theme
- * (all properties except size/strokeWidth and 'dark')
+ * Semantic stroke width tokens for icons
  */
-type ExtractColorTokens<T> = Exclude<
-  keyof T,
-  `size${string}` | `strokeWidth${string}` | 'dark'
->
+type IconStrokeWidthToken = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | '2xl'
 
-type IconSizeToken = ExtractSizeTokens<NewComponentTypes['Icon']>
-type IconStrokeWidthToken = ExtractStrokeWidthTokens<NewComponentTypes['Icon']>
-type IconColorToken = ExtractColorTokens<NewComponentTypes['Icon']>
+/**
+ * Semantic color tokens from Icon theme
+ */
+type IconColorToken =
+  | 'baseColor'
+  | 'mutedColor'
+  | 'successColor'
+  | 'errorColor'
+  | 'warningColor'
+  | 'infoColor'
+  | 'onColor'
+  | 'inverseColor'
+  | 'disabledBaseColor'
+  | 'disabledOnColor'
+  | 'dark'
+  | 'ai' // symbolic token for AI gradient colors, this does not exist in the theme
+  | 'navigationPrimaryBaseColor'
+  | 'navigationPrimaryHoverColor'
+  | 'navigationPrimaryActiveColor'
+  | 'navigationPrimaryOnColorBaseColor'
+  | 'navigationPrimaryOnColorHoverColor'
+  | 'navigationPrimaryOnColorActiveColor'
+  | 'actionSecondaryBaseColor'
+  | 'actionSecondaryHoverColor'
+  | 'actionSecondaryActiveColor'
+  | 'actionSecondaryDisabledColor'
+  | 'actionStatusBaseColor'
+  | 'actionStatusHoverColor'
+  | 'actionStatusActiveColor'
+  | 'actionStatusDisabledColor'
+  // | 'actionAiSecondaryTopGradientBaseColor' internally used for AI gradient
+  // | 'actionAiSecondaryBottomGradientBaseColor' internally used for AI gradient
+  | 'actionAiBaseColor'
+  | 'actionAiHoverColor'
+  | 'actionAiActiveColor'
+  | 'actionAiDisabledColor'
+  | 'actionPrimaryBaseColor'
+  | 'actionPrimaryHoverColor'
+  | 'actionPrimaryActiveColor'
+  | 'actionPrimaryDisabledColor'
+  | 'actionPrimaryOnColorBaseColor'
+  | 'actionPrimaryOnColorHoverColor'
+  | 'actionPrimaryOnColorActiveColor'
+  | 'actionPrimaryOnColorDisabledColor'
+  | 'accentBlueColor'
+  | 'accentGreenColor'
+  | 'accentRedColor'
+  | 'accentOrangeColor'
+  | 'accentGreyColor'
+  | 'accentAshColor'
+  | 'accentPlumColor'
+  | 'accentVioletColor'
+  | 'accentStoneColor'
+  | 'accentSkyColor'
+  | 'accentHoneyColor'
+  | 'accentSeaColor'
+  | 'accentAutoraColor'
+  | 'actionTertiaryBaseColor'
+  | 'actionTertiaryHoverColor'
+  | 'actionTertiaryActiveColor'
+  | 'actionTertiaryDisabledColor'
+  | 'actionSuccessSecondaryBaseColor'
+  | 'actionSuccessSecondaryDisabledColor'
+  | 'actionDestructiveSecondaryBaseColor'
+  | 'actionDestructiveSecondaryDisabledColor'
+  | 'actionAiSecondaryDisabledColor'
+  | 'actionSecondaryOnColorBaseColor'
+  | 'actionSecondaryOnColorHoverColor'
+  | 'actionSecondaryOnColorActiveColor'
+  | 'actionSecondaryOnColorDisabledColor'
+  | 'actionSuccessSecondaryHoverColor'
+  | 'actionSuccessSecondaryActiveColor'
+  | 'actionDestructiveSecondaryHoverColor'
+  | 'actionDestructiveSecondaryActiveColor'
 
 type InstUIIconOwnProps = {
   /**
```

Comment:
Again, I can't comment on unchanged parts, but:
StorkeWidth is a not allowed prop, all stroke width have to be  calculated from size and are not changable.
Size van not be number, only the preset tokens. 
Same for color. No inherit, no string, only the presets
Ask about these from desing